packages/now-cli/src/util/get-files.ts

```diff
@@ -1,7 +1,7 @@
-import { resolve, join } from 'path';
```

Comment:
Except for the removal of the `getAllProjectFiles()` function, the rest of the changes in this file are only cosmetic.